composer.json

```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": "^7.1",
         "ext-mbstring": "*",
-        "php-http/client-common": "^1.7",
+        "php-http/client-common": "^1.9 || ^2.0",
```

Comment:
No BC break to handle for us?

Reply:
Nothing detected with the existing tests.